classes/Cart.php

```diff
@@ -697,10 +697,11 @@ public function getProducts($refresh = false, $id_product = false, $id_country =
         $products_ids = array();
         $pa_ids = array();
         if ($result) {
+            $id_zone = Country::getIdZone($id_country);
```

Comment:
Should I change all the variable names to camelcase or just the ones introduced by me?

Reply:
Should I change the new SpecificPrice's property id_zone to idZone? This will affect the table too. Is that what you want by saying 

> (I won't spam the whole commit changes, but this is available everywhere you introduce a new class property, method, variable, parameter, etc.)

?

Reply:
Start with the ones you introduced. Renaming everything would be a pain, and would imply lots more tests to validate the PR.
About the table indexes, maybe we should stay consistent and stick to the snake case...